Ginger/Ginger/Run/RunSetsExecutionsHistoryPage.xaml.cs

```diff
@@ -396,19 +735,29 @@ private void DeleteExecutionReports(System.Collections.IList runSetReports)
                 }
             }
 
-            if (grdExecutionsHistory.Grid.SelectedItems.Count > 0)
+            if (xGridExecutionsHistory.Grid.SelectedItems.Count > 0)
             {
-                LoadExecutionsHistoryData();
+                RefreshDataAsync();
             }
         }
 
         private void RefreshGrid(object sender, RoutedEventArgs e)
         {
-            ReloadData();
+            RefreshDataAsync();
         }
 
         private void GetExecutionResultsFolder(object sender, RoutedEventArgs e)
         {
+            foreach (RunSetReport runSetReport in xGridExecutionsHistory.Grid.SelectedItems)
```

Comment:
Use radio button to check if Remote is selected and display the message accordingly